english/marshmallow.md

```diff
@@ -1,20 +1,22 @@
 Once upon a time many authors collaborated to make meaningless linked story.
 
-[start](mad/mad.md) your journey into world of mad 
+[start](mad/mad.md) your journey into world of mad
 
 or
 
 spend your time [wisely](http://www.pythonchallenge.com/)
 
 [Click here](http://www.dvloper.me) to visit my dvloper.me blog .
 
-Your adventure begins. You realize there are hundreds of paths for you to choose from. 
-Which way should you go? Only you can decide. 
+Your adventure begins. You realize there are hundreds of paths for you to choose from.
+Which way should you go? Only you can decide.
 
 Oh so many decisions - what will you do??
 
 You decide to drop everything you're doing and [run out the door](run_out_the_door/run_out_the_door.md)!
 
+Or instead of dropping everything, you can lift things to greater heights with [powerlifting](powerlifting/powerlifting.md)!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.